modules/services/sxhkd.nix

```diff
@@ -0,0 +1,85 @@
+{config, lib, pkgs, ...}:
+
+with lib;
+
+let
+
+  cfg = config.services.sxhkd;
+
+  hotkeybindingsStr = concatStringsSep "\n" (
+    mapAttrsToList (hotkey: command:
+      optionalString (command != null) ''
+        ${hotkey}
+          ${command}
+      ''
+    )
+    cfg.hotkeybindings
+  );
+
+in
+
+{
+  options.services.sxhkd = {
+    enable = mkEnableOption "Simple X hotkey daemon";
+
+    package = mkOption {
```

Comment:
I would suggest to remove this option and simply using `pkgs.sxhkd` directly. A user wanting to override the package can always use overlays or `packageOverrides`.